avogadro/qtplugins/simuneantinput/simuneantinput.h

```diff
@@ -0,0 +1,82 @@
+/******************************************************************************
+
+  This source file is part of the Avogadro project.
+
+  Copyright 2012 Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+
+#ifndef AVOGADRO_QTPLUGINS_QUANTUMINPUT_H
+#define AVOGADRO_QTPLUGINS_QUANTUMINPUT_H
+
+#include <avogadro/qtgui/extensionplugin.h>
+
+class QAction;
+class QDialog;
+
+namespace MoleQueue {
+class JobObject;
+}
+
+namespace Avogadro {
+namespace Io {
+class FileFormat;
+}
+
+namespace QtPlugins {
+class GamessHighlighter;
+
+class SimuneAntInputDialog;
+
+class SimuneAntInput : public QtGui::ExtensionPlugin
+{
+  Q_OBJECT
+
+public:
+  explicit SimuneAntInput(QObject *parent = 0);
+  ~SimuneAntInput();
+
+  QString name() const { return tr("GAMESS input"); }
+
+  QString description() const { return tr("Generate input for GAMESS."); }
```

Comment:
Same here - please take a bit of time to clean up the descriptions so people know what they're reading. Thanks!

Reply:
Thank you for your responses.

These days I'm going to defend my PhD thesis, so I have just no time, but as soon as I can I will solve all the issues.

Sorry for all the bugs in the code.